charters/web5-working-group.md

```diff
@@ -0,0 +1,133 @@
+# Web5 Working Group Charter
+
+**Start Date**: June 14, 2024
+
+**End Date**: December 31, 2024 
+
+**DRI**: [Frank Hinek](https://github.com/frankhinek)
+
+## Motivation and Background
+
+[Web5](https://developer.tbd.website/projects/web5/) is defined as...
+
+> A decentralized web platform that puts you in control of your data and identity.
+
+> The web democratized the exchange of information, but it's missing a key layer: identity. We struggle to secure personal data with hundreds of accounts and passwords we can’t remember. On the web today, identity and personal data have become the property of third parties.
+
+> Web5 brings decentralized identity and data storage to your applications. It lets devs focus on creating delightful user experiences, while returning ownership of data and identity to individuals.
+
+TBD has been building Web5 (nearly) since its inception. Web5 is the foundational layer of the TBD stack upon which decentralized applications, protocols, and services can be built. Today, Web5 is a collection of projects that are built on open standards pertaining to decentralized identity and self-sovereign technologies like Decentralized Identifiers (DIDs), and Verifiable Credentials (VCs), and Decentralized Web Nodes (DWNs) for secure personal storage and decentralized protocols.
+
+A key driver for this initiative is tbDEX, a use case that exemplifies the potential of the Web5 technology stack. tbDEX leverages Web5 to enable secure, decentralized value exchanges without intermediaries. This project highlights the practical applications and benefits of Web5 technologies, serving as a proof of concept that will guide the development and adoption of similar decentralized applications.
+
+Through this Working Group, we aim to continue building a robust and scalable framework for Web5, providing developers with the tools they need to build innovative and secure decentralized applications. This Working Group will coordinate with related working groups that may break off to focus on specific components of Web5, or have dependencies on this group's work.
+
+## Scope
+
+* Selection of standards pertaining to digital identifiers, credentials, storage, and associated protocols.
+
+* Implementation of a common set of standards using a well-defined API for the usage of these standards across programming languages.
+
+* Support for the needs of the TBD business unit, including, but not limited to the feature set required by [tbDEX](https://developer.tbd.website/projects/tbdex/).
+
+* Conformance infrastructure such as test vectors, test suites to encourage compatible implementations across programming languages.
+
+* Enable the development of Decentralized Applications (dApps) and Protocols using technologies such as Decentralized Identifiers (DIDs) and Verifiable Credentials (VCs).
```

Comment:
Should DWNs be referenced here as well? Or would that be confusing with the line that references `standards across programming languages`?

Reply:
I will add that here.